content/riak/kv/2.2.6/developing/data-types/gsets.md

```diff
@@ -450,19 +440,19 @@ var_dump($gset->getData());
 ```
 
 ```python
-cities_set.dirty_value
+gset.dirty_value
 
 # The value fetched from Riak is always immutable, whereas the "dirty
 # value" takes into account local modifications that have not been
 # sent to the server. For example, where the call above would return
 # frozenset(['Toronto', 'Hamilton', 'Ottawa']), the call below would
 # return frozenset([]).
 
```

Comment:
The above text (lines 445-449) needs to be adjusted to the current banking context

Reply:
Adjusted to reflect the banking context